base_continent/__manifest__.py

```diff
@@ -1,42 +1,26 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Author: Romain Deheele
-#    Copyright 2014 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2014-2016 Camptocamp SA (Author: Romain Deheele)
```

Comment:
© is supposed to be replaced by the full word Copyright according to latest conventions (remark for all occurrences in all files)